org.eclipse.jdt.ls.target/org.eclipse.jdt.ls.tp.target

```diff
@@ -25,7 +25,7 @@
             <unit id="org.eclipse.sdk.feature.group" version="0.0.0"/>
             <unit id="org.mockito.mockito-core" version="0.0.0"/>
             <unit id="org.apache.commons.commons-io" version="0.0.0"/>
-            <repository location="https://download.eclipse.org/eclipse/updates/4.30-I-builds/I20231030-1800/"/>
+            <repository location="https://download.eclipse.org/eclipse/updates/4.31-I-builds/I20231203-1800/"/>
```

Comment:
You can probably get rid of this. https://github.com/eclipse-jdtls/eclipse.jdt.ls/pull/2994 is merged.